src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
```

Comment:
The usage of tag names for styling is not recommended in CSS. It's better to style elements using `.class`. This is a code style issue. You should style elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.